src/modules/paymentProcessor/bridging/ConnextBridgeLogic.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+import {CrosschainBase_v1} from
+    "src/modules/paymentProcessor/bridging/abstracts/CrosschainBase_v1.sol";
+import {IERC20PaymentClientBase_v1} from
+    "@lm/interfaces/IERC20PaymentClientBase_v1.sol";
+import {IERC20} from "@oz/token/ERC20/IERC20.sol";
+
+interface IEverclearSpoke {
+    function newIntent(
+        uint32[] memory destinations,
+        address to,
+        address inputAsset,
+        address outputAsset,
+        uint amount,
+        uint24 maxFee,
+        uint48 ttl,
+        bytes memory data
+    ) external returns (bytes32 intentId, uint amountOut);
+}
+
+contract ConnextBridgeLogic is CrosschainBase_v1 {
+    IEverclearSpoke public everClearSpoke;
+    address public immutable weth;
+
+    constructor(address _everclearSpoke, address _weth) CrosschainBase_v1() {
+        everClearSpoke = IEverclearSpoke(_everclearSpoke);
+        weth = _weth;
+    }
+
+    function xcall(
+        IERC20PaymentClientBase_v1.PaymentOrder memory order,
+        bytes memory executionData
+    ) external payable returns (bytes32 intentId) {
+        // Decode any additional parameters from executionData
+        (uint maxFee, uint ttl) = abi.decode(executionData, (uint, uint));
```

Comment:
Do we need to validate if the values are valid, i.e. are we able to find out if something would revert on the bridge side given something went wrong with the value configuration?

Reply:
Yes 100% we do, we need to check that ttl is not zero and the fee isn't greater than max fee. Nice point out, ideally we're also testing for this and the unhappy paths.

Reply:
Addressed in new PR.